eng/scripts/verify-npm-tags.ps1

```diff
@@ -0,0 +1,59 @@
+param (
+  [Parameter(mandatory = $true)]
+  $artifactName,
+  [Parameter(mandatory = $true)]
+  $workingDirectory
+)
+
+# TODO: delete testing comments below
+#'@azure/eventhubs-checkpointstore-table' alpha, no beta, no GA
+#'@azure/openai' alpha, beta, no GA
+#'@azure/video-indexer-widgets' no alpha, no beta, GA
+#'@azure/template' alpha, beta, GA
+#$pkgProps.PackageId
+
+. (Join-Path $PSScriptRoot "../common/scripts/common.ps1")
+
+$packageProperties = Get-PkgProperties -PackageName $artifactName
+$packageName =  $packageProperties.Name
+Write-Host "Verify npm tag versions for package $packageName"
+$packageVersions = npm view $packageName versions --json | ConvertFrom-Json
+$validDev = $packageVersions | ? { $_ -match "alpha" } | Select-Object -Last 1
+$validNext = $packageVersions | ? { $_ -match "beta" } | Select-Object -Last 1
+$validLatest = $packageVersions | ? { !($_ -match "alpha") -and !($_ -match "beta") } | Select-Object -Last 1
+if (!$validLatest) {
+  Write-Host "No GA package found"
+  $validLatest = $packageVersions | ? { $_ -match "beta" } | Select-Object -Last 1
+}
+if (!$validLatest) {
+  Write-Host "No beta package found"
+  $validLatest = $packageVersions | ? { $_ -match "alpha" } | Select-Object -Last 1
+}
+
+$currentDev = npm view $packageName@dev version --silent
```

Comment:
Why do we need to make multiple calls? Can we not get everything from one `npm view @azure/openai --json`?

Reply:
fixed~